mesonbuild/compilers/d.py

```diff
@@ -332,7 +332,7 @@ def _translate_args_to_nongnu(self, args: T.List[str]) -> T.List[str]:
                 else:
                     dcargs.append('-I' + arg[10:])
                 continue
-            elif arg.startswith('-L/') or arg.startswith('-L./'):
+            elif arg.startswith('-L'):
```

Comment:
A search path can start with anything, not only `/` and `./`. On Windows, it cannot start with `/`.